src/Uno.UI/UI/Xaml/Controls/CalendarView/CalendarViewBaseItemChrome.uno.cs

```diff
@@ -91,7 +91,7 @@ private void UpdateChrome()
 				// We don't support inner border yet, so even if not optimal we just use it as border.
 				borderBrush = selectedBrush;
 			}
-
+			BorderBrush = borderBrush;
```

Comment:
What effect does this have? Is it related to the linked bug?

Reply:
However, even if the CalendarViewItem's state (Selected, Today, etc.) changes (i.e. the border changes), the BorderBrush does not change.

Reply:
I think this is expected

Reply:
So you mean BorderBrush must not be changed?